addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -77,6 +77,7 @@ export class LinkPopover extends Component {
             buttonSize: this.props.linkEl.className.match(/btn-(sm|lg)/)?.[1] || "",
             buttonStyle: this.initButtonStyle(this.props.linkEl.className),
             isImage: this.props.isImage,
+            isLabelHidden: !!this.props.linkEl.childElementCount,
```

Comment:
Rename to `showLabel` and reverse the value.

Reply:
Adapted and repusheddd